ccd-definition/ComplexTypes/CareSupervision/GeneratedOrder.json

```diff
@@ -61,6 +61,17 @@
     "Comment": "During event, document is linked to page 1 for visibility on check answers page - during tab view stage, make document visible as part of Order object",
     "CategoryID": "orders"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "GeneratedOrder",
+    "ListElementCode": "documentConfidential",
+    "FieldType": "Document",
+    "ElementLabel": "Confidential Order document",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "date=\"*\" OR dateTimeIssued=\"*\"",
+    "Comment": "During event, document is linked to page 1 for visibility on check answers page - during tab view stage, make document visible as part of Order object",
+    "CategoryID": "ordersConfidential"
+  },
```

Comment:
Do we need a translatedDocumentConfidential? Perhaps we need another another ticket to cater this.

Reply:
good question, may need a new ticket for the "Upload translated document" event